app/views/student_task/list.html.erb

```diff
@@ -46,6 +46,22 @@
     </span><br/>
   <% end %>
 
+    <strong>&nbsp;&nbsp;<span class="tasknum">&nbsp;<%= @latetasks.size.to_s %>&nbsp;</span> Late Tasks<br></strong><br>
+     <% @latetasks.each do |student_task|
```

Comment:
Write the loop in view like this is really weird, had better put it in `student_task_helper.rb`.